Userguides: Update restore from keys (#2387)
monero-project/monero-site: _i18n/en/resources/user-guides/restore_from_keys.md

Launch `monero-wallet-gui`. If this is the first time you launched it go to the next step, otherwise click `Cancel`:

![cancel opening](/img/resources/user-guides/en/restore_from_keys/cancel-opening.avif)

Click on `Restore wallet from keys or mnemonic seed`:

![choose restore](/img/resources/user-guides/en/restore_from_keys/choose-restore.avif)

> Select `Restore from keys`, give your wallet a name & storage location, and complete the `Account address (public)`, `View key (private)` & `Spend key (private)`. Optionaly, specify a `Restore height (optional)` to avoid scanning of oldest blocks. Then click the `Next` button:

Reviewer: ```suggestion
Select `Restore from keys`, give your wallet a name & storage location, and complete the `Account address (public)`, `View key (private)` & `Spend key (private)`. Optionally, specify a `Restore height (optional)` to avoid scanning of oldest blocks. Then click the `Next` button:
```

![restore wallet](/img/resources/user-guides/en/restore_from_keys/restore-wallet.avif)

On the next page, give your wallet a strong password and confirm it before clicking the `Next` button:

![wallet password](/img/resources/user-guides/en/restore_from_keys/wallet-password.avif)

Specify your daemon settings and click the `Next` button:

![daemon settings](/img/resources/user-guides/en/restore_from_keys/daemon-settings.avif)

> Click on `Create wallet` to enjoy your restored wallet:

Reviewer: ```suggestion
Click on `Create wallet` to complete the restoration process:
```
